stories/general/EmptyState.stories.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * TEAM: frontend_infra
+ *
+ * @flow
+ */
+
+import * as React from "react";
+import {StyleSheet, css} from "aphrodite";
+import {storiesOf} from "@storybook/react";
+import {text, withKnobs} from "@storybook/addon-knobs";
+import sections from "../sections";
+import Button from "../../button/Button";
+import EmptyState from "../../EmptyState";
+import type {GraphicIcons} from "../../tools/graphicIcons";
+
+const stories = storiesOf(`${sections.interaction}/EmptyState`, module);
+stories.addDecorator(withKnobs);
+stories.add("Empty State", () => (
+  <div className={css(styles.container)}>
+    <EmptyStateHoist {...getEmptyStateKnobs()} />
+  </div>
+));
+
+type Props = {|
+  +icon: GraphicIcons,
+  +headline: string,
+  +body: string,
+|};
+
+function EmptyStateHoist({icon, headline, body}: Props) {
```

Comment:
NIT: Instead of duplicating the definition for Props, you can use `React.ElementConfig<typeof EmptyState>` to compute EmptyState's Props definition. Or you could also use `*` to infer Props based on `EmptyStateHoist`'s usage.

```suggestion
function EmptyStateHoist({icon, headline, body}: *) {
```